server/data/words/german.json

```diff
@@ -404,9 +404,13 @@
     "translation": "always"
   },
   {
-    "word": "tolle",
+    "word": "toll",
```

Comment:
I could see on Google translate that tolle is the translate community approved German word for amazing and toll is the translate community approved German word for great.

Reply:
you are absolutely right, if i give a little context to the word "tolle" you absolutely can translate it to great.
would it be possible to habe both words in there? If not i would suggest to keep "toll" because "tolle" on its own is kinda tricky because as you can se with my suggestion for quiff, just make it upercase and you get the quiff :)